### PR TITLE
feat: APP-508 show book call button on terrasos project bottom banner

### PR DESCRIPTION
**Pull request by blushi**
## Description

https://regennetwork.atlassian.net/browse/APP-508

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [x] provided a link to the relevant issue or specification
- [x] provided instructions on how to test
- [x] reviewed "Files changed" and left comments if necessary
- [x] confirmed all CI checks have passed

### How to test

1. From Terrasos app https://deploy-preview-2565--terrasos.netlify.app/project/biocultural-jaguar-credits-ancestral-stewardship-in-the-sharamentsa-community-2, "Book a call" button for terrasos projects going to terrasos website contact page
2. "Book a call" button from compliance info tabs is the same https://deploy-preview-2565--terrasos.netlify.app/project/C02-009
3. From Regen app https://deploy-preview-2565--regen-marketplace.netlify.app/project/354637ec-a687-11ef-b1b2-022e215a294f, "Book a call" button for terrasos projects going to internal calendy
4. From Regen app, other off chain projects remains without bottom banner https://deploy-preview-2565--regen-marketplace.netlify.app/project/woodburn

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

File: web-marketplace/sanity-codegen.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 overwrite: true
-schema: 'https://jm12rn9t.api.sanity.io/v1/graphql/staging/default'
+schema: 'https://jm12rn9t.api.sanity.io/v1/graphql/staging/book-call'
```
Comment (blushi) [inline web-marketplace/sanity-codegen.yml:2]: TODO revert before merging

**Comment by blushi** [comment]
@erikalogie @S4mmyb see testing instructions

**Comment by S4mmyb** [comment]
LGTM

**Comment by erikalogie** [comment]
LGTM!

File: web-marketplace/src/components/templates/ProjectDetails/ProjectDetails.tsx [hunk 2]
```diff
@@ -338,6 +351,8 @@ function ProjectDetails(): JSX.Element {
       projectBySlug?.data.projectBySlug?.published;
   const projectLocation = projectMetadata?.['schema:location'];
 
+  const isTerrasos = normalizedProject.type === 'TerrasosProjectInfo';
```
Comment (r41ph) [inline web-marketplace/src/components/templates/ProjectDetails/ProjectDetails.tsx:354]: can we use the `IS_TERRASOS` constant from `lib/env` in this case?

Comment (blushi) [reply web-marketplace/src/components/templates/ProjectDetails/ProjectDetails.tsx:354]: no this isn't the same thing
IS_TERRASOS just means it's the terrasos app, which allows us to use their custom theme and features as on https://terrasos.app.regen.network/
(basically it's just `VITE_MARKETPLACE_CLIENT` env var that we need to set to tell the app to run as regen marketplace or terrasos app)
whereas here, we're checking whether a particular project is a terrasos project or not, and we checj the `@type` value that is stored in the project metadata

**Review by r41ph — APPROVED** [review]

**Pull request merged**